statvar_imports/ndap/nss_health/pvmap/metadata_seven.csv

```diff
@@ -0,0 +1,17 @@
+parameter,value

```

Comment:
not clear what the '_seven' means in the filename,
Please name this file so that users knows which download this applies to, such as the metadata_dataset_7314.csv